### ISSUE TITLE
[Broken Website] websites with beige backgrounds turn orange

### ISSUE THREAD

**Issue by mournfully** [darkreader/darkreader#10288]
# Broken Website Report

## Website Address

https://news.ycombinator.com/
https://ghostwriter.kde.org/

## Steps To Reproduce

- Go to either of those websites with Dark Reader enabled

## Problematic Observation

I don't think dark reader is supposed to turn beige backgrounds into dark-green or orange, and I'm not sure if it's just me but it looks kinda bad. Expected behaviour was that Dark Reader would turn the text white and the background dark-grey or something along those lines.

## Screenshots

![screenshot](https://user-images.githubusercontent.com/98147018/201399840-b9aedf29-8e2c-49c4-9a2f-35f4d9818c8b.png)
![screenshot](https://user-images.githubusercontent.com/98147018/201399925-3b2d0366-cc4e-4f9e-960c-a6abb831d5f4.png)

## System Information

- OS: Void Linux
- Browser: Version 107.0.5304.87 (Official Build) (64-bit)
- Dark Reader Version: 4.9.60

**Comment by Gusted** [comment]
> I don't think dark reader is supposed to turn beige backgrounds into dark-green or orange, and I'm not sure if it's just me but it looks kinda bad. Expected behaviour was that Dark Reader would turn the text white and the background dark-grey or something along those lines.

Yeah that's the actual behavior in a lot of cases, but in both cases, the site is slightly yellow(ghostscript more noticeable), which is translated into dark yellow by dark reader as it tries to preserve the original colors of the site.

**Comment by mournfully** [comment]
> Yeah that's the actual behavior in a lot of cases, but in both cases, the site is slightly yellow(ghostscript more noticeable), which is translated into dark yellow by dark reader as it tries to preserve the original colors of the site.

Fair enough, but are there any plans to change this sort of behaviour or add a setting to overwrite original colour schemes by making the background-colour property on these elements transparent? I'm getting tired of having to manually tweak css for so many websites...

**Comment by Gusted** [comment]
No there are currently no plans to change that behavior or have a setting for it.

**Comment by mournfully** [comment]
Fair enough. But, if I were _add predefined CSS snippets to devtools that affect only the current website_, would that get merged? For example, if you hovered over a button in a dropdown menu it would show a preview and when clicked it would add a snippet like the ones below to devtools. I'm sure if this is a feature worth adding though, hence the question.
```
[data-darkreader-inline-bgcolor] {
    background-color: transparent !important; 
}
```
```
html, body {
    background-color: #181a1b !important; 
}
```

If such a feature couldn't be merged, do know if it's possible to interact with darkreader from the cli? 
Using `grep -rl "go\.dev" ~/.config/chromium/` (a website I've manually added to devtools that wasn't there before). I found mentions of it in `~/.config/chromium/Default/Session Storage/000862.ldb`. But, I'm not sure if modifying it directly is such a good idea. 

So, basically do you know if it's possible to interact with darkreader from cli? Cause I'm much more comfortable with just making a script to automate most of this, compared to local development with npm and js and then having to maintain that fork.

**Comment by Gusted** [comment]
> Fair enough. But, if I were to make a menu that could _add predefined CSS snippets to devtools that affect only the current website_, would that get merged?

I'm very conservative of adding new features. As people really expect first-class support and integration -- But mainly, I prefer to not get removed CWS for adding "unrelated" features (which for some absurd reasons have been reasons to remove extensions from CWS). For such purposes, I really can recommend [Stylus](https://github.com/openstyles/stylus).

> So, basically do you know if it's possible to interact with darkreader from cli? Cause I'm much more comfortable with just making a script to automate most of this, compared to local development with npm and js and then having to maintain that fork.

If you're willing to go trough a lot of loops, likely it's possible, but I don't really know 🤷🏽.

**Comment by mournfully** [comment]
@Gusted Thank you so much for your help. Stylus was exactly what I needed. Although, Dark Reader does tend to try and mess with my changes if I don't use the "not inverted site list". 

On that note, do you know if it's possible to import/export Dark Reader's "Site List". I'd like to be able to back it up, instead of manually entering each url one by one.

**Comment by mournfully** [comment]
> On that note, do you know if it's possible to import/export Dark Reader's "Site List". I'd like to be able to back it up, instead of manually entering each url one by one.

Never mind, I figured it out from this [github issue](https://github.com/darkreader/darkreader/issues/4362).
> Click Dark Reader icon.
> Click Dev tools (in the bottom-right corner).
> Click Preview new design.
> Close DevTools
> Click Dark Reader icon.
> Press Settings
> Press Manage settings
> Click on export
> You can import from there also

I'll close this issue now, thank you again for all your help.
